### DISCUSSION TITLE
`overrides.channels.email.action_url` does not work

### DISCUSSION THREAD

**Discussion by 3v0k4** [magicbell/community#65] [Questions] [answered]
The [Channel Overrides](https://www.magicbell.com/docs/overrides#channel-overrides) documentation states that `email.action_url` should work. But, when I pass `overrides.channels.email.action_url`, the email notification uses the non-overridden `action_url` passed to POST /notifications.

Tested with both the default email provider (Ping) and Sendgrid.

[Support ticket](https://magicbell.supportbee.com/tickets/40588994).

## Repro steps:

- add below email template
- send below notification

**email template**

```
action_url: {{ notification.action_url }}
```

**notification**

```bash
curl https://api.magicbell.com/notifications \
  --request POST \
  --header 'accept: application/json' \
  --header 'content-type: application/json' \
  --header 'X-MAGICBELL-API-SECRET: MY_API_SECRET' \
  --header 'X-MAGICBELL-API-KEY: MY_API_KEY' \
  --data '{
    "notification": {
        "title": "test overrides.channels.email.action_url",
        "content": "testcontent",
        "category": "announcement",
        "action_url": "https://magicbell.com/docs",
        "recipients": [{ "email": MY_EMAIL }],
        "overrides": {
          "channels": {
            "email": {
              "action_url": "https://example.com"
            }
          }
        }
    }
  }'
```

**Comment by smeijer** [comment]
Sorry about that. We've confirmed the bug, and will get this fixed. As it's already mid December, we hope to push a fix early next year.

**Comment by unamashana** [comment, marked as answer]
Please see #64 for some workarounds.
